test/tests/phpmyadmin-apache-run/phpmyadmin_test.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import re
+import sys
+
+import mechanize
+import tempfile
+import pytest
+
+
+def do_test_content(match, content):
+    assert(match in content)
+
+def test_phpmyadmin(url, username, password, server, sqlfile):
```

Comment:
I recently pushed more changes upstream
And started to try to plug them in on our CI
But the having to start a mysql server blocked me

I think you can pick my changes on the py test script :)